### PR TITLE
Sample Import: add taxon as an optional command param

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Sample/Command/Import/Base.t [hunk 1]
```diff
@@ -4,10 +4,28 @@ use strict;
...
+    is => 'Genome::Sample::Command::Import::Base',
+    has => {
+        nomenclature => {},
+        # these are deprecated
+        name_regexp => { default_value => '(TGI\-[\w\d]+)\-[\w\d]+', },
+    },
+};
+
+my $taxon = Genome::Taxon->__define__(name => 'almost human');
+ok($taxon, 'defined taxon');
```
Comment (tmooney) [inline lib/perl/Genome/Sample/Command/Import/Base.t]: Is this `$taxon` meant to be used in this test?

Comment (ebelter) [inline lib/perl/Genome/Sample/Command/Import/Base.t]: It is. It is/was/going to be included in another PR to be used further in the test. I will remove it.

**Comment by tmooney** [comment]
+1 :monkey_face:

**Comment by acoffman** [comment]
+1 :cop:

**Comment by ebelter** [comment]
Had to manually merge.

**Pull request merged**